src/main/java/crazypants/enderio/machine/obelisk/GuiRangedObelisk.java

```diff
@@ -72,7 +73,7 @@ protected void drawGuiContainerBackgroundLayer(float par1, int par2, int par3) {
     drawCenteredString(fontRenderer, EnderIO.lang.localize("gui.spawnGurad.range") + " " + range, getGuiLeft() + sx / 2 + 9, getGuiTop() + 68,
         ColorUtil.getRGB(Color.white));
   }
-
+  
```

Comment:
please set your "save actions" to remove whitespace at the end of lines. Thx.

I also have it set to reformat changed lines, which I find to be a good idea, too.